ansible/Dockerfile

```diff
@@ -1,14 +1,9 @@
-FROM alpine:3.8
+FROM python:2.7-alpine3.8
 
-ARG CI_HELPER_VERSION
-ARG PYTHON_VERSION
-ARG VERSION
+ARG VERSION="2.5.5"
```

Comment:
Should the version have a default value?

Reply:
Deleted